appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -388,7 +388,7 @@ private void addDayActivityDetailLink(DisclosureResponseMessageDto message)
 
 		private void addDayActivityDetailLink(GoalConflictMessageDto message)
 		{
-			String dateStr = DayActivityDto.formatDate(message.getActivityStartTime().toLocalDate());
+			String dateStr = DayActivityDto.formatDate(message.getStartDate());
```

Comment:
The idea is to add a link to the day activity linked to the message, which is not the same (logically) as the date of the message itself, is it?